components/page-templates/B2B/ListsTemplate/ListTemplate.styles.tsx

```diff
@@ -0,0 +1,22 @@
+export const styles = {
+  addNewListButtonStyles: {
```

Comment:
Button styles are already handled globally. Don't change in component level

Reply:
done